src/sparsey/add_remove.rs

```diff
@@ -0,0 +1,28 @@
+use sparsey::prelude::*;
+
+struct A(f32);
+struct B(f32);
+
+pub struct Benchmark(World, Vec<Entity>);
+
+impl Benchmark {
+    pub fn new() -> Self {
+        let mut world = World::default();
+        world.register::<A>();
+        world.register::<B>();
+
+        let entities = world.create_entities((0..1_000).map(|_| (A(0.0),))).to_vec();
```

Comment:
This should be 10_000.

Reply:
Thanks! I might have overwritten a zero when I added the underscores.